src/components/ui/Select/Styled.tsx

```diff
@@ -13,8 +17,11 @@ export const StyledSelectInput = styled.select`
   line-height: ${props => props.theme.fontSizes.text.lineHeight};
   height: 2rem;
   letter-spacing: -0.005625rem;
-  padding: 0.375rem 0.5rem;
-  transition: box-shadow .05s ease-in;
+  width: 100%;
+  padding: 0.375rem 1.5rem 0.375rem 0.5rem;
+  transition: box-shadow 0.05s ease-in;
+  display: inline-block;
+  appearance: none;
```

Comment:
I'm assuming that this handles removing default styling across our supported browsers? Browsers render this element in a variety of different ways. I've built custom `select`s before, and getting it right for a bunch of browsers was always a challenge.

Reply:
Good point! I did tested on iOS14, MacOs (FF, Safary, Chrome) and Windows (Edge, FF, Chrome) will also ask QA to verify once merged.